rockcraft.yaml

```diff
@@ -8,38 +8,36 @@ base: ubuntu:22.04
 platforms:
   amd64:
 
+run_user: _daemon_
+
+environment:
+  SPARK_HOME: /opt/spark
+  SPARK_CONFS: /etc/spark8t/conf
+  JAVA_HOME: /usr/lib/jvm/java-11-openjdk-amd64
+  PYTHONPATH: /opt/spark/python:/opt/spark8t/python/dist:/usr/lib/python3/dist-packages
+  PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/spark:/opt/spark/bin:/opt/spark/python/bin:/opt/spark-client/python/bin
+  HOME: /home/spark
```

Comment:
this is interestingly problematic

Reply:
Very good point. I'm actually not sure what to do here. In `/home/spark/` we are currently not storing anything, but we place here stuff like the .kube folder when using the image in a pod. 

In the new setup, we actually don't have any other home folder anymore (`_daemon_` does not come with a home) if we remove this

Reply:
changed HOME directory to `/var/lib/spark`